js/ui/gallery.js

```diff
@@ -37,6 +37,7 @@ const GALLERY_INDICATOR_CLASS = GALLERY_CLASS + '-indicator';
 const GALLERY_INDICATOR_ITEM_CLASS = GALLERY_INDICATOR_CLASS + '-item';
 const GALLERY_INDICATOR_ITEM_SELECTOR = '.' + GALLERY_INDICATOR_ITEM_CLASS;
 const GALLERY_INDICATOR_ITEM_SELECTED_CLASS = GALLERY_INDICATOR_ITEM_CLASS + '-selected';
+const ITEM_CONTENT_CLASS = '.dx-item-content';
```

Comment:
It is not a class. It is a selector. Could you change a variable name or value like this:

`const ITEM_CONTENT_SELECTOR` or `'dx-item-content'`